src/app/pages/TopicPage/TopicGrid/index.jsx

```diff
@@ -52,13 +59,23 @@ const Wrapper = styled.div``;
 
 const TopicGrid = ({ promos }) => {
   const { dir } = useContext(ServiceContext);
+  const hasMultiplePromos = promos.length > 1;
+  const firstPromo = promos[0];
   return (
     <Wrapper>
-      {promos.map(promo => (
-        <Item key={promo.id} dir={dir}>
-          <TopicPromo {...promo} />
+      {hasMultiplePromos ? (
+        <TopicList role="list">
```

Comment:
Should we be adding unit tests for both these paths?

Reply:
can do. i've just been testing manually but can also add in unit tests as the other pages have that have the same sort of logic :)